_data/site_menu.yml

```diff
@@ -70,8 +70,94 @@ administration:
       desc: "Viewing and collecting information from your clusters, hosts, virtual machines, and other resources"
 
     - title: Managing Providers
-      path: "/docs/reference/latest/managing_providers/index.html"
-      desc: "Managing your infrastructure, cloud, and containers providers and datastores"
+      children:
+        - title: Overview
+          path: "/docs/reference/latest/managing_providers/index.html"
+          desc: "Managing your infrastructure, cloud, and containers providers and datastores"
+
+        - title: Red Hat Virtualization Providers
```

Comment:
@Fryguy I was thinking of having these under e.g. `Infrastructure Providers`  sections but that'd be an extra level

Reply:
to be accurate, it should be under the `Virtualization Infrastructure Providers` section (as opposed to `Physical Infrastructure Providers`).  I know that we use `Infrastructure Providers` in code and docs, but I believe that is a shorthand for `Virtualization Infrastructure Providers`

Reply:
We call it `Infrastructure` and `Physical Infrastructure` in the UI, while I agree `Virtual Infrastructure` is more accurate I think the docs should match the UI so people aren't confused looking for the `Virtual Infrastructure` section in the UI
![Screenshot from 2021-12-03 09-01-21](https://user-images.githubusercontent.com/12851112/144615187-a1881f6a-a5e9-4915-bb5f-f3ca6753f299.png)

Reply:
wondering if we should rename UI to have `Virtual Infrustructure` next to `Physical Infrastructure`

Reply:
:+1: yeah I'd be for that

Reply:
@Fryguy what do you think?  And this suggestion is not necessarily for this PR

Reply:
Yeah I like "Virtual Infrastructure" in the UI and in the docs